### DISCUSSION TITLE
request help: How to get the kubeconfig file outside k3s server or by token

### DISCUSSION THREAD

**Discussion by gewei2000** [k3s-io/k3s#3487] [Q&A]
How to get the kubeconfig file outside k3s server or by token

**Comment by JaccoVeldscholten** [comment]
Use SFTP to retreive the file.

**Comment by remkolems** [comment]
1. Use https://github.com/subhra74/snowflake
2. Go to Files tab
3. Download file at /etc/rancher/k3s/k3s.yaml to another location

or

1. sudo nano /etc/rancher/k3s/k3s.yaml
2. select all and ctrl + shift + c
3. paste contents to new file on target computer/software
